Core/scripts/dirac-install.py

```diff
@@ -1021,296 +1053,313 @@ def fixBuildPaths():
-      setattr( cliParams, opName, [ opV.strip() for opV in opVal.split( "," ) if opV ] )
-
-  #Now parse the ops
+    if isinstance(getattr(cliParams, opName), basestring):
+      setattr(cliParams, opName, opVal)
+    elif isinstance(getattr(cliParams, opName), bool):
+      setattr(cliParams, opName, opVal.lower() in ("y", "yes", "true", "1"))
+    elif isinstance(getattr(cliParams, opName), list):
+      setattr(cliParams, opName, [opV.strip() for opV in opVal.split(",") if opV])
+
+  # Now parse the ops
   for o, v in optList:
-    if o in ( '-r', '--release' ):
+    if o in ('-r', '--release'):
       cliParams.release = v
-    elif o in ( '-l', '--project' ):
+    elif o in ('-l', '--project'):
       cliParams.project = v
-    elif o in ( '-e', '--extensions' ):
-      for pkg in [ p.strip() for p in v.split( "," ) if p.strip() ]:
+    elif o in ('-e', '--extensions'):
+      for pkg in [p.strip() for p in v.split(",") if p.strip()]:
         if pkg not in cliParams.extensions:
-          cliParams.extensions.append( pkg )
-    elif o in ( '-t', '--installType' ):
+          cliParams.extensions.append(pkg)
+    elif o in ('-t', '--installType'):
       cliParams.externalsType = v
-    elif o in ( '-i', '--pythonVersion' ):
+    elif o in ('-i', '--pythonVersion'):
       cliParams.pythonVersion = v
-    elif o in ( '-p', '--platform' ):
+    elif o in ('-p', '--platform'):
       cliParams.platform = v
-    elif o in ( '-d', '--debug' ):
+    elif o in ('-d', '--debug'):
       cliParams.debug = True
-    elif o in ( '-g', '--grid' ):
+    elif o in ('-g', '--grid'):
       cliParams.lcgVer = v
-    elif o in ( '--no-lcg-bundle' ):
+    elif o in ('--no-lcg-bundle'):
       cliParams.noLcg = True
-    elif o in ( '-u', '--baseURL' ):
+    elif o in ('-u', '--baseURL'):
       cliParams.installSource = v
-    elif o in ( '-P', '--installationPath' ):
+    elif o in ('-P', '--installationPath'):
       cliParams.targetPath = v
       try:
-        os.makedirs( v )
+        os.makedirs(v)
       except:
         pass
-    elif o in ( '-v', '--useVersionsDir' ):
+    elif o in ('-v', '--useVersionsDir'):
       cliParams.useVersionsDir = True
-    elif o in ( '-b', '--build' ):
+    elif o in ('-b', '--build'):
       cliParams.buildExternals = True
-    elif o in ( "-B", '--noAutoBuild' ):
+    elif o in ("-B", '--noAutoBuild'):
       cliParams.noAutoBuild = True
-    elif o in ( '-X', '--externalsOnly' ):
+    elif o in ('-X', '--externalsOnly'):
       cliParams.externalsOnly = True
-    elif o in ( '-T', '--Timeout' ):
+    elif o in ('-T', '--Timeout'):
       try:
-        cliParams.timeout = max( cliParams.timeout, int( v ) )
-        cliParams.timeout = min( cliParams.timeout, 3600 )
+        cliParams.timeout = max(cliParams.timeout, int(v))
+        cliParams.timeout = min(cliParams.timeout, 3600)
       except ValueError:
         pass
-
+    elif o in ('-O', '--diracos-version'):
```

Comment:
and here too, I would put 
`elif o == 'dirac-os-version'`

Reply:
done